src/Testing/StaticFixtureProvider.php

```diff
@@ -14,8 +14,10 @@ public static function yieldFilesFromDirectory(string $directory, string $suffix
     {
         $fileInfos = self::findFilesInDirectory($directory, $suffix);
 
+        $key = 0;
         foreach ($fileInfos as $fileInfo) {
-            yield $fileInfo->getFilename() => [$fileInfo->getPathName()];
+            yield "{$key} " . $fileInfo->getFilename() => [$fileInfo->getPathName()];
+            $key++;
```

Comment:
Why not use key of this foreach?

Reply:
key contains the filepath

Reply:
Ah, I used `FinderSanitizer` to remove this WTF

Ok